VideoWeb/VideoWeb.UnitTests/Controllers/ParticipantController/GetTestCallResultForParticipantTests.cs

```diff
@@ -63,7 +63,7 @@ public async Task Should_return_ok_when_testcall_result_returned()
                 .Returns(Task.FromResult(testCallResponse));
 

```

Comment:
can you please revert the white space changes.

Reply:
n/a